include/CppUTest/CppUTestConfig.h

```diff
@@ -216,8 +218,6 @@ typedef struct { char dummy[16]; } cpputest_ulonglong;
 typedef struct { char dummy[8]; } cpputest_longlong;
 typedef struct { char dummy[8]; } cpputest_ulonglong;
 #endif
-#define CPPUTEST_LONGLONG_DEFAULT cpputest_longlong()
-#define CPPUTEST_ULONGLONG_DEFAULT cpputest_ulonglong()
```

Comment:
Declaring a variable of type `cpputest_[u]longlong` is fine as long as it's used exclusively in tests where `CPPUTEST_USE_LONG_LONG` is undefined.  It's more dodgy in the other case.  Having said that, I tried to not use the "default" value where real longlongs were available, so it might be ok.